src/PatientChecking/PatientChecking/Services/ServiceModels/PatientCheckInContext.cs

```diff
@@ -25,10 +25,7 @@ public PatientCheckInContext(DbContextOptions<PatientCheckInContext> options)
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            if (!optionsBuilder.IsConfigured)
-            {
-               optionsBuilder.UseSqlServer("Data Source=5CD1199VTK;Initial Catalog=PatientCheckIn;Integrated Security=True");
```

Comment:
don't use PC name "5CD1199VTK", you can use Data Source=(local) so all local machines can use it

Reply:
fixed as per reviewer request